react/features/feedback/components/FeedbackDialog.web.js

```diff
@@ -0,0 +1,342 @@
+import StarIcon from '@atlaskit/icon/glyph/star';
+import StarFilledIcon from '@atlaskit/icon/glyph/star-filled';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { Dialog } from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import JitsiMeetJS from '../../base/lib-jitsi-meet';
+
+import { cancelFeedback, submitFeedback } from '../actions';
+
+declare var interfaceConfig: Object;
+
+const scoreAnimationClass = interfaceConfig.ENABLE_FEEDBACK_ANIMATION
+    ? 'shake-rotate' : '';
+
+/**
+ * The scores to display for selecting. The score is the index in the array and
+ * the value of the index is a translation key used for display in the dialog.
+ *
+ * @types {string[]}
+ */
+const SCORES = [
+    'feedback.veryBad',
+    'feedback.bad',
+    'feedback.average',
+    'feedback.good',
+    'feedback.veryGood'
+];
+
+/**
+ * A React {@code Component} for displaying a dialog to rate the current
+ * conference quality, write a message describing the experience, and submit
+ * the feedback.
+ *
+ * @extends Component
+ */
+class FeedbackDialog extends Component {
+    /**
+     * {@code FeedbackDialog} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The cached feedback message, if any, that was set when closing a
+         * previous instance of {@code FeedbackDialog}.
+         */
+        _message: React.PropTypes.string,
+
+        /**
+         * The cached feedback score, if any, that was set when closing a
+         * previous instance of {@code FeedbackDialog}.
+         */
+        _score: React.PropTypes.number,
+
+        /**
+         * The JitsiConference for which the feedback will be about.
+         *
+         * @type {JitsiConference}
+         */
+        conference: React.PropTypes.object,
```

Comment:
Can you add a comment here, explaining that we need to pass the conference as a prop in order to access it after we left. I first got confused why we're not just getting it from the store.